pages/_app.tsx

```diff
@@ -0,0 +1,32 @@
+import Head from 'next/head';
+import type { AppProps } from 'next/app';
+
+import { Box, FullHeight, ThemeProvider } from '@smg-automotive/components';
+
+import { Brand } from 'types/brand';
+
+import Fonts from '../components/Fonts';
+import ErrorBoundary from '../components/ErrorBoundary';
```

Comment:
I sugges rolling without any error handling (500/404) as it is something we can follow up on in the technical interview, ie. how do you handle errors in a react application?